gempak/ush/gfs_meta_ver.sh

```diff
@@ -1,372 +1,65 @@
-dc25=$($NDATE -150 ${PDY}${cyc} | cut -c -10)
-date25=$(echo ${dc25} | cut -c -8)
-sdate25=$(echo ${dc25} | cut -c 3-8)
-cycle25=$(echo ${dc25} | cut -c 9,10)
-# DEFINE 26 CYCLES AGO
-dc26=$($NDATE -156 ${PDY}${cyc} | cut -c -10)
-date26=$(echo ${dc26} | cut -c -8)
-sdate26=$(echo ${dc26} | cut -c 3-8)
-cycle26=$(echo ${dc26} | cut -c 9,10)
-# DEFINE 27 CYCLES AGO
-dc27=$($NDATE -162 ${PDY}${cyc} | cut -c -10)
-date27=$(echo ${dc27} | cut -c -8)
-sdate27=$(echo ${dc27} | cut -c 3-8)
-cycle27=$(echo ${dc27} | cut -c 9,10)
-# DEFINE 28 CYCLES AGO
-dc28=$($NDATE -168 ${PDY}${cyc} | cut -c -10)
-date28=$(echo ${dc28} | cut -c -8)
-sdate28=$(echo ${dc28} | cut -c 3-8)
-cycle28=$(echo ${dc28} | cut -c 9,10)
-# DEFINE 29 CYCLES AGO
-dc29=$($NDATE -174 ${PDY}${cyc} | cut -c -10)
-date29=$(echo ${dc29} | cut -c -8)
-sdate29=$(echo ${dc29} | cut -c 3-8)
-cycle29=$(echo ${dc29} | cut -c 9,10)
-# DEFINE 30 CYCLES AGO
-dc30=$($NDATE -180 ${PDY}${cyc} | cut -c -10)
-date30=$(echo ${dc30} | cut -c -8)
-sdate30=$(echo ${dc30} | cut -c 3-8)
-cycle30=$(echo ${dc30} | cut -c 9,10)
-# DEFINE 31 CYCLES AGO
-dc31=$($NDATE -192 ${PDY}${cyc} | cut -c -10)
-date31=$(echo ${dc31} | cut -c -8)
-sdate31=$(echo ${dc31} | cut -c 3-8)
-cycle31=$(echo ${dc31} | cut -c 9,10)
-# DEFINE 32 CYCLES AGO
-dc32=$($NDATE -204 ${PDY}${cyc} | cut -c -10)
-date32=$(echo ${dc32} | cut -c -8)
-sdate32=$(echo ${dc32} | cut -c 3-8)
-cycle32=$(echo ${dc32} | cut -c 9,10)
-# DEFINE 33 CYCLES AGO
-dc33=$($NDATE -216 ${PDY}${cyc} | cut -c -10)
-date33=$(echo ${dc33} | cut -c -8)
-sdate33=$(echo ${dc33} | cut -c 3-8)
-cycle33=$(echo ${dc33} | cut -c 9,10)
+ln -s "${COM_ATMOS_GEMPAK_1p00}" "${RUN}.${PDY}${cyc}"
+export COMIN="${RUN}.${PDY}${cyc}"
 
 # SET CURRENT CYCLE AS THE VERIFICATION GRIDDED FILE.
-vergrid="F-${MDL} | ${PDY2}/${cyc}00"
+vergrid="F-${MDL} | ${PDY:2}/${cyc}00"
 fcsthr="f00"
 
-# SET WHAT RUNS TO COMPARE AGAINST BASED ON MODEL CYCLE TIME.
-#if [ ${cyc} -eq 00 ] ; then
-#    verdays="${dc1} ${dc2} ${dc3} ${dc4} ${dc5} ${dc6} ${dc7} ${dc8} ${dc9} ${dc10} ${dc11} ${dc12} ${dc13} ${dc14} ${dc15} ${dc16} ${dc17} ${dc18} ${dc19} ${dc20} ${dc21}"
-#elif [ ${cyc} -eq 12 ] ; then
-#    verdays="${dc1} ${dc2} ${dc3} ${dc4} ${dc5} ${dc6} ${dc7} ${dc8} ${dc9} ${dc10} ${dc11} ${dc12} ${dc13} ${dc14} ${dc15} ${dc16} ${dc17} ${dc18} ${dc19} ${dc20} ${dc21}"
-#else
-#    verdays="${dc1} ${dc2} ${dc3} ${dc4} ${dc5} ${dc6} ${dc7} ${dc8} ${dc9} ${dc10} ${dc11} ${dc12} ${dc13} ${dc14} ${dc15} ${dc16} ${dc17} ${dc18} ${dc19} ${dc20} ${dc21}"
-#fi 
-
-verdays="${dc1} ${dc2} ${dc3} ${dc4} ${dc5} ${dc6} ${dc7} ${dc8} ${dc9} ${dc10} ${dc11} ${dc12} ${dc13} ${dc14} ${dc15} ${dc16} ${dc17} ${dc18} ${dc19}
-${dc20} ${dc21} ${dc22} ${dc23} ${dc24} ${dc25} ${dc26} ${dc27} ${dc28} ${dc29} ${dc30} ${dc31} ${dc32} ${dc33}"
+MDL2="GFSHPC"
+#GENERATING THE METAFILES.
+# seq won't give us any splitting problems, ignore warnings
+# shellcheck disable=SC2207
+IFS=$'\n' lookbacks=($(seq 6 6 198))
```

Comment:
I think this should be:
`IFS=$'\n' lookbacks=($(seq 6 6 180) $(seq 192 12 216))`

Reply:
Fixed